src/components/shared/custom-modal-window/style.module.scss

```diff
@@ -0,0 +1,24 @@
+.modal_window {
```

Comment:
is your styles applying for a mui component?

Reply:
yes